plugins/digitalocean/resoto_plugin_digitalocean/resources.py

```diff
@@ -139,16 +143,16 @@ class DigitalOceanKubernetesCluster(DigitalOceanResource, BaseResource):
 
     kind: ClassVar[str] = "digitalocean_kubernetes_cluster"
 
-    version: str = field(default="")
-    cluster_subnet: str = field(default="")
-    service_subnet: str = field(default="")
-    ipv4: str = field(default="")
-    endpoint: str = field(default="")
-    auto_upgrade: bool = field(default=False)
-    status: str = field(default="")
-    surge_upgrade: bool = field(default=False)
-    registry_enabled: bool = field(default=False)
-    ha: bool = field(default=False)
+    do_k8s_version: str = field(default="")
```

Comment:
Only information, no change required:
if you only have a default value, you can write: `do_k8s_version: str = ""`.
Using field allows for more data to attach.
Example: `do_k8s_version: str = field(default="", metadata={"description":"Description of the field"})`
Would also export field level documentation to the core.